source/compilation/builtins/SystemTasks.cpp

```diff
@@ -730,6 +782,21 @@ void registerSystemTasks(Compilation& c) {
     TASK("$q_full", SubroutineKind::Function, 1, 1);
 
 #undef TASK
+
+#define PLA_TASK(name) c.addSystemSubroutine(std::make_unique<PlaTask>(name))
+    std::vector<std::string> arrayType = {"$async", "$sync"};
```

Comment:
Not that it matters a whole lot, but since these are fixed size I'd use std::array